src/rad/resources/schemas/l1_detector_guidewindow-1.0.0.yaml

```
@@ -0,0 +1,1408 @@
+            ndim: 2
+      min_bind_flag:
+        title: Minimum Binding Flag
+        description: |
+          Either a 2D array containing the binding minimum flag for each pedestal
+          and signal resultant or a single integer if all flags are equal.
+          1 indicates that any negative pixel values within the resultant were bound to 0.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: int16
+            exact_datatype: true
+            ndim: 2
+      max_bind_flag:
+        title: Maximum Binding Flag
+        description: |
+          Either a 2D array containing the binding maximum flag for each pedestal
+          and signal resultant or a single integer if all flags are equal.
+          1 indicates that any positive pixel values within the resultant were bound to 65535.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: int16
+            exact_datatype: true
+            ndim: 2
+      # Optional
+      exposure_mapping:
+        title: Exposure Mapping
+        description: |
+          1D array of signed integers with the same length as the tracking signal resultants.
+          Positive integers correspond to the exposure number (single MA Table read).
+          Negative integers correspond to the tracking reads between exposures.
+        ndim: 1
+        datatype: uint16
+        exact_datatype: true
+
+  # Optional
+  edge_acq_data:
+    title: Spectrum Edge Acquisition Guide Window Data
+    description: |
+      All data received from the spectrum edge acquisition guide window data packets during
+      WSM exposures. This secondary acquisition only occurs during spectroscopic observations.
+    type: object
+    properties:
+      pedestal_resultants:
+        title: Pedestal Resultants (DN)
+        description: |
+          3D array containing every pedestal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      signal_resultants:
+        title: Signal Resultants (DN)
+        description: |
+          3D array containing every signal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      lower_left_corners:
+        title: Lower Left Corners
+        description: |
+          2D array containing all the lower left corners of each guide
+          window resultant pair in science coordinates (x_start, y_start).
+        ndim: 2
+        datatype: uint16
+        exact_datatype: true
+      upper_right_corners:
+        title: Upper Right Corners
+        description: |
+          2D array containing all the upper right corners of each guide
+          window resultant pair in science coordinates (x_stop, y_stop).
+        ndim: 2
```

```suggestion
        tag: tag:stsci.edu:asdf/core/ndarray-1.*
        ndim: 2
```
Missing `tag` keyword.